src/components/ComponentView.js

```diff
@@ -67,7 +67,7 @@ export const ComponentView = ({
   componentMeta,
   ocmRepo,
 }) => {
-  const [componentDescriptor, isLoading, isError, error] = useFetchComponentDescriptor({
+  const [componentDescriptor, state] = useFetchComponentDescriptor({
     componentName: componentMeta.name,
     ocmRepoUrl: ocmRepo,
     version: componentMeta.version,
```

Comment:
Parameter names must also be changed accordingly (-> `ocmRepo` and `componentVersion`) (same for other usages).